src/lib/components/TokenSelect/TokenButton.tsx

```diff
@@ -36,7 +36,7 @@ interface TokenButtonProps {
 export default function TokenButton({ value, collapsed, disabled, onClick }: TokenButtonProps) {
   return (
     <StyledTokenButton onClick={onClick} empty={!value} color={value ? 'interactive' : 'accent'} disabled={disabled}>
```

Comment:
Can you pull all these colors out into a useMemo? Seems redundant and confusing with all the ternaries.